ci/functional/job_cleanup.sh

```diff
@@ -8,9 +8,10 @@ rm -rf install/lib/daos/TESTING/ftest/avocado/job-results/job-*/html/
 
 # Remove the latest avocado symlink directory to avoid inclusion in the
 # jenkins build artifacts
-unlink install/lib/daos/TESTING/ftest/avocado/job-results/latest
-rm -rf "Functional/"
-mkdir "Functional/"
+if [ -e install/lib/daos/TESTING/ftest/avocado/job-results/latest ]; then
+  unlink install/lib/daos/TESTING/ftest/avocado/job-results/latest
```

Comment:
Honestly, I'm not sure.  The '-e' test should detect if the file exists regardless of type (one could argue '-L' may make sense here, given the action taken if true).  Launch.py relies heavily on the existence of the 'latest' directory - without it we would not have the 'daos_logs' and 'daos_configs' directories and their files in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3190/16/artifact/Functional/job-2020-08-13T03.39-bde3265-daos_test-daos_core_test/.  So we have evidence that it existed and by virtue of it not existing in artifact/Functional we know it was removed (or someow otherwise not included in the [move](https://github.com/daos-stack/daos/blob/master/ci/functional/job_cleanup.sh#L24).

Reply:
Ah, could it be https://github.com/daos-stack/daos/blob/master/ci/functional/job_cleanup.sh#L19-20?  Since it explicitly looks for "job-*" it would exclude "latest".

Reply:
Still doesn't explain why the '-e' doesn't detect it though.

Reply:
Indeed.  It appears benign.  I just don't like not understanding why things that should be are not since the lack of understanding could be papering over other issues.

It doesn't help that `Jenkinsfile` size limitations have forced us into this popcorn-trail of indirections in following the flow of a test run any more.  :-(

Reply:
I agree completely about not liking why the issue is not understood.  I am also questioning the need for the unlink command if we ignore the directory when moving the the test results to the 'Functional' directory.

Reply:
I don't think the uncomfortableness outweighs the need to have other PRs correctly detect Functional HW Medium test failures.

Reply:
True.  But how do we avoid this just being forgotten and/or never addressed as it _looks_ benign?  Even if we open a ticket on it, it will never get any attention due to it's apparent benign-ness.  Until some other silent failure highlights it.  :-(